mofacts/client/views/experiment/card.js

```diff
@@ -372,7 +372,16 @@ Template.card.rendered = async function() {
   }
 
   // Check if TDF allows for dialogue feedback preferences, if so, route to dialogue feedback widget
-  if(Session.get('allowFeedbackTypeSelect') && !Session.get('feedbackParamsSet')) {
+  let allowFeedbackTypeSelect = false;
+  let feedbackUnitNumLookup = 0;
+
+  if(typeof Session.get('currentUnitNumber') !== 'undefined'){
```

Comment:
currentUnitNumber is only going to be set somewhat into card.js, in resumeComponentState.  At that point the deliveryParams have also been read in and you can use getCurrentDeliveryParams.  Can you refactor this to happen at the end of resumeFromComponentState by jumping to the feedback page and doing a check similar to the end of Template.card.rendered but with a bit to then go to processUserTimesLog if we jumped out to feedback before doing the syllable cache check and it?